pkg/monitor/format/format.go

```diff
@@ -249,6 +251,9 @@ func (m *MonitorFormatter) FormatSample(data []byte, cpu int) {
 		m.policyVerdictEvents(prefix, data)
 	case monitorAPI.MessageTypeRecCapture:
 		m.recorderCaptureEvents(prefix, data)
+	case monitorAPI.MessageTypeTraceSock:
+		log.Infof("sock-traces-debug %+v %v", data, messageType)
+		fmt.Printf("%s Sock event: %+v\n", prefix, data)
```

Comment:
I think the `data` probably needs to be properly formatted somehow? Looks like it's just `[]byte` at this point, which will not be very user-friendly.